src/SDDK/type_definition.hpp

```diff
@@ -0,0 +1,21 @@
+/** \file linalg.hpp
+ *
+ *  \brief Define some common type used in SDDK::
+ */
+
+#ifndef SIRIUS_LINALG_BASE_H
```

Comment:
```
#ifndef __TYPE_DEFINITIONS_HPP__
...
```
and also the documentation section for the header is wrong

Reply:
I haven't add the remaining copyright header as I am not sure what is the year to put in